### ISSUE TITLE
Deploy Solution not working

### ISSUE THREAD

**Issue by fab2903** [MicrosoftDocs/architecture-center#2556]
The link to Github to deploy the solution is broken. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 49d69a9c-86d9-c2bf-de02-ae2dc851a086
* Version Independent ID: a17ffde4-5e77-2749-05c9-b5ec7340b7bf
* Content: [Extend an on-premises network using VPN - Azure Architecture Center](https://docs.microsoft.com/en-us/azure/architecture/reference-architectures/hybrid-networking/vpn)
* Content Source: [docs/reference-architectures/hybrid-networking/vpn.yml](https://github.com/microsoftdocs/architecture-center/blob/master/docs/reference-architectures/hybrid-networking/vpn.yml)
* Service: **architecture-center**
* Sub-service: **reference-architecture**
* GitHub Login: @doodlemania2
* Microsoft Alias: **pnp**

**Comment by BhargaviAnnadevara-MSFT** [comment]
@fab2903 Thank you for catching that error in the docs. I have submitted a PR to fix the issue, the change is merged internally and will go-live in the public docs shortly.

**Comment by neilpeterson** [comment]
@fab2903 thanks for opening this issue. We recently migrated to a new 'browser' for the AAC content which required significant file refactoring, a few links were inadvertently updated incorrectly.

@BhargaviAnnadevara-MSFT thanks for addressing this one. Prior to seeing this issue, I had also submitted this PR to fix many similar across the repo:

https://github.com/MicrosoftDocs/architecture-center-pr/pull/2206
